lectures/linear_equations.md

```diff
@@ -668,25 +734,32 @@ $$
 ```{math}
 :label: two_matrix
 
-p = (C - D)^{-1} h.
+    p = (C - D)^{-1} h.
```

Comment:
@longye-tian these indentations shouldn't be needed. Do you add them for style?

Reply:
I'll change it back. I think I put too much space after the : label: